test/integration/collada_world_exporter.cc

```diff
@@ -110,6 +110,42 @@ TEST_F(ColladaWorldExporterFixture, ExportWorldFromFuelWithSubmesh)
   common::removeAll(outputPath);
 }
 
+TEST_F(ColladaWorldExporterFixture, ExportWorldMadeFromObj)
+{
+  ignition::common::setenv("IGN_GAZEBO_RESOURCE_PATH",
+    (std::string(PROJECT_SOURCE_PATH) + "/test/worlds:" +
```

Comment:
Please use [ignition::common::JoinPaths](https://ignitionrobotics.org/api/common/4.0/namespaceignition_1_1common.html#aa64aaa25ed8793c66bd8d8f4062c2109) here and below

Reply:
fixed a475ee4